src/main/java/spark/SparkBase.java

```diff
@@ -241,6 +249,38 @@ public static synchronized void externalStaticFileLocation(String externalFolder
     }
 
     /**
+     * Maps the given path to the given WebSocket handler.
+     * <p>
+     * This is currently only available in the embedded server mode.
+     *
+     * @param path the WebSocket path.
+     * @param handler the handler class that will manage the WebSocket connection to the given path.
+     */
+    public static synchronized void webSocket(String path, Class<?> handler) {
```

Comment:
are the no restrictions on class type here (essentially in WebSocketCreatorFactory.create(webSocketHandlers.get(path))?